traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -309,6 +311,22 @@ func GetLDAPConfig(LDAPConfPath string) (bool, *ConfigLDAP, error) {
 	if err != nil {
 		return false, LDAPconf, fmt.Errorf("parsing LDAP conf '%v': %v", LDAPConfBytes, err)
 	}
+	if strings.TrimSpace(LDAPconf.AdminPass) == "" {
+		return false, LDAPconf, fmt.Errorf("LDAP conf missing admin_pass field")
+	}
+	if strings.TrimSpace(LDAPconf.SearchBase) == "" {
+		return false, LDAPconf, fmt.Errorf("LDAP conf missing search_base field")
+	}
+	if strings.TrimSpace(LDAPconf.AdminDN) == "" {
+		return false, LDAPconf, fmt.Errorf("LDAP conf missing admin_dn field")
+	}
+	if strings.TrimSpace(LDAPconf.Host) == "" {
+		return false, LDAPconf, fmt.Errorf("LDAP conf missing host field")
+	}
+	if strings.TrimSpace(LDAPconf.SearchQuery) == "" {
+		return false, LDAPconf, fmt.Errorf("LDAP conf missing search_query field")
+	}
+
```

Comment:
shouldn't the trimmed values be what ends up being stored in LDAPconf?   Otherwise, the caller needs to also deal with any spaces...